src/NFTAuction.sol

```diff
@@ -75,15 +56,26 @@ contract NFTAuctionMachine is
             address(this)
         )
     {
-        if (_duration == 0) {
```

Comment:
might we need this back?

Reply:
just remove it to save gas since if we deploy we can assume it won't be 0 but we can add back

Reply:
@nnnnicholas removed it to save gas since if we deploy we can assume it won't be 0 wdyt ?